src/infra/ucan.js

```diff
@@ -5,80 +5,114 @@ const ucans = require('ucans');
 
 const serviceDID = config.SERVICE_DID;
 
-let verify = (req, res, next) => {
-  req.requestId = uuidv4();
-  console.log('req.requestId: ', req.requestId);
-  let token = req.headers['authorization']; // Express headers are auto converted to lowercase
-  if (token && token.startsWith('Bearer ')) {
-    // Remove Bearer from string
-    token = token.slice(7, token.length);
+
+async function namespaceValidation(namespace, invokerAddress, token) {
```

Comment:
may be rename to `validateNamespace` same for all new new functions

Reply:
`validateContractAddress` and so on

Reply:
alright, doing it.